kuma/settings/common.py

```diff
@@ -1203,16 +1197,15 @@ def pipeline_one_scss(slug, **kwargs):
 
 if MAINTENANCE_MODE:
     # In maintenance mode, we're going to avoid using the database, and
-    # use Celery's default beat-scheduler as well as memcached for storing
+    # use Celery's default beat-scheduler as well as Redis for storing
     # any results. In both normal and maintenance mode we use djcelery's
     # loader (see djcelery.setup_loader() above) so we, among other things,
     # acquire the Celery settings from among Django's settings.
     CELERYBEAT_SCHEDULER = 'celery.beat.PersistentScheduler'
     DEFAULT_CELERY_RESULT_BACKEND = (
-        'cache+memcached://' + ';'.join(
-            config('MEMCACHE_SERVERS',
-                   default='127.0.0.1:11211',
-                   cast=Csv())
+        'redis://' + ';'.join(
+            config('REDIS_CACHE_SERVER',
+                   default='127.0.0.1:6379')
```

Comment:
This looks like it won't work, will return something like ``1;2;7;.;0;.0;...``. I'll need to try in ``MAINTENANCE_MODE`` to be sure. I also suspect we want to continue using the default rather than override.

Reply:
Confirmed that this results in ``DEFAULT_CELERY_RESULT_BACKEND=redis://r;e;d;i;s;:;/;/;r;e;d;i;s;:;6;3;7;9;/;3``. It should be:

```
      DEFAULT_CELERY_RESULT_BACKEND = REDIS_CACHE_SERVER
```